app/client/src/components/pages/Community.Tabs.DrinkingWater.js

```diff
@@ -826,6 +909,34 @@ function DrinkingWater() {
                                 </tr>
                               )}
                             </tbody>
+                            {tribalWithdrawerCount > 0 && (
```

Comment:
Same as above. The Tribal Status switch should always be visible and just disabled if the `tribalWithdrawerCount` equals 0.

Reply:
Sounds good, yeah I wasn't sure about that.

Reply:
Thanks for making that change.